static/styles/portico/landing_page.css

```diff
@@ -4004,6 +4000,10 @@ nav {
             hsl(49, 71%, 68%) 80%
         );
     }
+
+    .features-page .gradients .gradient {
+        height: 700px;
+    }
```

Comment:
I think you can just reference `.portico-landing.features-app` -- that's the existing class for this purpose.

Reply:
Or is the issue that the gradients are included outside that block?  IF so, could we just move them inside?

Reply:
Yes, the gradients block is outside the block. Done.